app/controllers/users_controller.rb

```diff
@@ -0,0 +1,29 @@
+class UsersController < ApplicationController
+	skip_before_action :ensure_current_user
```

Comment:
Usual ruby style is to use 2 spaces for tabs, rather than tab characters. 

Have a look at 
https://github.com/thoughtbot/guides/tree/master/style/ruby
https://github.com/airbnb/ruby
https://github.com/bbatsov/rubocop